modules/dpu-pxe-boot-additional-hosts.adoc

```diff
@@ -0,0 +1,142 @@
+
+.Procedure
+
+
+. Clone the link:https://github.com/rh-ecosystem-edge/ansible-ai-playbook[Ansible Assisted Installer Playbooks] repo.
++
+[NOTE]
+======
+Clone the repo on the installer node or on your laptop (that has access to the installer node).
+======
++
+[source,terminal]
+----
+$ git clone https://github.com/rh-ecosystem-edge/ansible-ai-playbook.git
+----
+
+. Edit the `playbook.yaml`.
++
+[source,yaml]
+----
+---
+- hosts: all
+  roles:
+    - setup-pxe <1>
+    - download-iso-pxe
+  vars:
+    ARCH: arm
+    URL: http://<URL>:<PORT> <2>
+    CLUSTER_ID: "<CLUSTER UUID>" <3>
+----
++
+<1> This field sets up the basic services required for PXE booting
+<2> Enter the Assisted Installer URL and port number
+<3> Enter the cluster UUID from the Assisted Installer UI.
++
+[NOTE]
+====
+The option exists to download the ISO manually and extract what is needed to support PXE booting the hosts. To do this add two variables to the `playbook.yaml`. The variables are:
+
+* ISO_NAME: "discovery_image.iso" - This variable is the name of the downloaded ISO.
+* WORKDIR: "/tmp/download-iso-pxe"- This variable specifies the directory where the ISO is downloaded to.
+====
+
+. Download and extract the discovery ISO by using the following command.
++
+[source,terminal]
+----
+$ ansible-playbook -i localhost, playbook.yaml --tags extract,download
+----
++
+[NOTE]
+====
+Use `localhost` if running the command on the installer node. Replace `localhost` with the `FQDN` of the installer node if running the command on your laptop. By default the name of the downloaded ISO is `discovery_image.iso` and it is downloaded to `/tmp/download-iso-pxe`.
+====
+
+. Verify the discovery image exists in `/tmp/download-iso-pxe`.
+
+.. List the contents of the download directory.
++
+[source,terminal]
+----
+$ ll /tmp/download-iso-pxe
+----
++
+.Expected output
++
+[source,terminal]
+----
+discovery_image.iso
+----
+
+. Verify the correct files are extracted to the correct locations.
+
+.. Verify the ignition file and rootfs image exists in `/var/www/html/pxe`.
++
+[source,terminal]
+----
+$ ll /var/www/html/pxe
+----
++
+.Expected output
++
+[source,terminal]
+----
+total 839720
+-rw-r--r--. 1 apache apache     10484 Mar 23 06:15 config.ign
+-rw-r--r--. 1 apache apache 859858944 Mar 23 06:15 rootfs.img
+----
+
+.. Verify the `grub.cfg` and the files needed to boot the VM exist in `/var/lib/tftpboot`.
++
+[source,terminal]
+----
+$ ll /var/lib/tftpboot
+----
++
+.Expected output
++
+[source,terminal]
+----
+total 89956
+-rw-r--r--. 1 root root   857984 Jan 28 01:13 BOOTAA64.EFI
+-rw-r--r--. 1 root root  2446280 Jan 28 01:13 grubaa64.efi
+-rw-r--r--. 1 root root      567 Mar 23 06:16 grub.cfg
+-rw-r--r--. 1 root root 79229244 Mar 23 06:06 initrd.img
+-rw-r--r--. 1 root root  9565104 Mar 23 06:06 vmlinuz
+----
+
+At this stage you have all the files required to PXE boot the DPU hosts.
```

Comment:
```suggestion
At this stage you have all the files required to PXE boot the DPU worker.
```

Reply:
This part only show get the pxe boot images by this ansible playbook,  but did not show how to use them and how to PXE boot the OS on BF cards,  I think it's not easy for customer to follow. cc @bn222 @wizhaoredhat @yingwang-0320